src/electron-main.ts

```diff
@@ -233,19 +233,10 @@ async function setupGlobals(): Promise<void> {
     const asarPath = await getAsarPath();
     await loadConfig();
 
-    // we assume the resources path is in the same place as the asar
-    const resPath = await tryPaths("res", path.dirname(asarPath), [
-        // If run from the source checkout
-        "res",
-        // if run from packaged application
-        "",
-    ]);
-
-    // The tray icon
-    // It's important to call `path.join` so we don't end up with the packaged asar in the final path.
```

Comment:
This comment was invalid, `path.dirname` above is actually what ditched the packages asar from the path